### ISSUE TITLE
Why there is "missing 'use strict'"

### ISSUE THREAD

**Issue by frogcjn** [palantir/tslint#1341]
In any typescript file, I wrote function definition:
```ts
function asdf() {
    console.log("here")
}
```
And then I got an warning from tslint
```
function
~~~~~~.  missing 'use strict'
```

But
```ts
const asdf = function() {
    console.log("here")
}
```

```ts
const asdf = () => {
    console.log("here")
}
```
Will not trigger warning.

**Comment by frogcjn** [comment]
Also see: https://github.com/palantir/tslint/issues/678

**Comment by adidahiya** [comment]
You should turn off the `use-strict` rule. See its [docs here](http://palantir.github.io/tslint/rules/use-strict/). I agree it's not a broadly useful rule, that's why we haven't enabled it in `tslint:recommended`.
